src/clj/fluree/db/json_ld/shacl.cljc

```diff
@@ -363,7 +367,7 @@
          [message]  const/sh_message
          expect     constraint}
         shape]
-    (cond-> {:subject (display focus-node)
+    (cond-> {:subject (display (first focus-node))
```

Comment:
I'd like to refactor this to use `where/TypedValue`, since it's the same basic abstraction. I'd like to do it as a separate PR, though, as it's a larger change.

Reply:
Ah. I thought you were saving that for later.

Reply:
Sorry, that's what I mean - I'm saving it for later. It's also weirdly similar to a `match` as well, and I'd like to spend some hammock time thinking about if these things are in fact the same thing and should be unified.